backend/src/nodes/utils/utils.py

```diff
@@ -454,3 +456,12 @@ def compare_conditions(a, b):
         raise RuntimeError(f"Unknown side selection {side}")
 
     return w_new, h_new
+
+
+def numerical_sort(value: str):
```

Comment:
Please add a return type. I had problems understanding this functions, and a return type would have helped.

```suggestion
def numerical_sort(value: str) -> List[Union[str, int]]:
```

Reply:
Whoops. Looks like there's another function that was added to utils without a return type too.

Reply:
Had to add a type: ignore. For some reason, pyright thinks the type should be have str | Any instead of str | int, but as far as I know those are the only two possible types.

Reply:
@theflyingzamboni What happens when you add `List[Union[str, int]]` to `parts`?

Reply:
Then the return value is fine, but it complains that list[str | Any] cannot be assigned to List[str | int].